### PR TITLE
Custom loss

### PR DESCRIPTION
**Pull request by henry090**

### CODE REVIEW

**Comment by henry090** [comment]
E.g. 

```
my_loss = nn_loss(function( inputs, targets, smooth=1L) {

  inputs = Functional$sigmoid(inputs)       
  

  inputs = inputs$view(-1L)
  targets = targets$view(-1L)
  
  intersection = (inputs * targets)$sum()                            
  dice = (2.*intersection + smooth)/(inputs$sum() + targets$sum() + smooth)  
  
  1L - dice
})
```
```
learn = Learner(dls, model, loss_func=my_loss, metrics=accuracy())
```

based on https://www.kaggle.com/bigironsphere/loss-function-library-keras-pytorch
